### DISCUSSION TITLE
(EDIT: Was not Frigate) Does Frigate do stuff at midnight? High CPU usage

### DISCUSSION THREAD

**Discussion by thuesen** [blakeblackshear/frigate#15331] [General]
I notice that at midnight, my CPU usage increases. When I access Frigate web UI, it says "CPU 74%" in the lower left corner, with an orange dot. Clicking it takes me to CPU usage page, but nothing in there indicates anything out of the ordinary, so I'm not sure why it's using such high CPU beginning at midnight and lasting for around two hours.  Any tips on how to investigate it?

**Comment by NickM-27** [comment]
Frigate builds the previews at the bottom of every hour, and there are some tasks done but never at midnight, they are definitely not long lasting tasks either

**Reply by thuesen** [reply]
It's 17 minutes past local midnight here now, and I notice it doing it again. You can see the CPU usage in Frigate UI showing 67% but no indication as to what is using it.

Not sure how to interpret this htop, if frigate.recording_manager is the culprit?

![Screenshot 2024-12-05 at 00 13 10](https://github.com/user-attachments/assets/2ddb9ffc-80a6-49d4-9a39-48599b1e60ee)

![Screenshot 2024-12-05 at 00 16 38](https://github.com/user-attachments/assets/076adb2e-8e87-414b-a3aa-130340258a0a)

**Reply by NickM-27** [reply]
No, nothing in Frigate is using enough CPU to be using 67% of all cores. This means it is likely something else running besides Frigate that is using this CPU

**Reply by thuesen** [reply]
I'm running Frigate in its own container on Proxmox, so I'd assume the 67% listed is related to that container and not the Home Assistant VM or my other container. But I don't know. I'll try to stop things and see when it ends :)

**Reply by thuesen** [reply]
Thanks for your time @NickM-27 (and sorry for wasting it), indeed it was _not_ Frigate's fault!

A backup provider was running on the host machine, but I only spotted increased CPU usage in the Frigate container so that's where I was digging down to begin with.
